### PR TITLE
Fixes out of bound exception during personal number validation

### LINKED ISSUE #137: We return 500 in case personal number 33 is sent to Backend

### PR DESCRIPTION
**Pull request by LukasForst**
* closes #137

### CODE REVIEW

File: backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt [hunk 1]
```diff
@@ -109,10 +109,10 @@ class PatientService(
 
         val (registration, registrationRemoteHost) = patientRegistrationDto
 
-        return PatientRegisteredDtoOut(newSuspendedTransaction {
-            logger.debug { "Registration validation for patient ${registration.email}." }
-            validationService.validatePatientRegistrationAndThrow(registration)
+        logger.debug { "Registration validation for patient ${registration.email}." }
```
Comment (marazt) [inline backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt:112]: for -> of

File: backend/src/main/kotlin/blue/mild/covid/vaxx/service/ValidationService.kt [hunk 2]
```diff
@@ -88,12 +89,12 @@ class ValidationService(private val questionService: QuestionService) {
         """(?:[a-z0-9!#${'$'}%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#${'$'}%&'*+/=?^_`{|}~-]+)*|"(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21\x23-\x5b\x5d-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])*")@(?:(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?|\[(?:(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9]))\.){3}(?:(2(5[0-5]|[0-4][0-9])|1[0-9][0-9]|[1-9]?[0-9])|[a-z0-9-]*[a-z0-9]:(?:[\x01-\x08\x0b\x0c\x0e-\x1f\x21-\x5a\x53-\x7f]|\\[\x01-\x09\x0b\x0c\x0e-\x7f])+)\])""".toRegex() matches email
 
     private fun validatePersonalNumber(personalNumber: String): Boolean {
-        if (personalNumber.isNullOrEmpty()) {
+        if (personalNumber.trim().length < 9) { // but can be bigger because of spaces
```
Comment (marazt) [inline backend/src/main/kotlin/blue/mild/covid/vaxx/service/ValidationService.kt:92]: nejprve bych to trimnul a pak pouzival to trimnute - na radku 92 a 99, kde to splitujes, at je to osetrene.

Comment (marazt) [reply backend/src/main/kotlin/blue/mild/covid/vaxx/service/ValidationService.kt:92]: Btw, napisme na to uz testy...

Comment (LukasForst) [reply backend/src/main/kotlin/blue/mild/covid/vaxx/service/ValidationService.kt]: ano testy by byly nice asi tak na vsechno, je na to task..

**Pull request merged**